src/azul/openapi/schema.py

```diff
@@ -138,7 +140,9 @@ def array(item: TYPE, *items: TYPE, **kwargs):
     return array_type(make_type(item), *map(make_type, items), **kwargs)
 
 
-def enum(*items: PrimitiveJSON, type_: TYPE = None) -> JSON:
+def enum(*items_: PrimitiveJSON,
+         items: Union[Sequence[str], Mapping] = None,
```

Comment:
Mapping from what to what?

Reply:
Changed due to removal of `mapping` support. #2375